src/jiraApi.gs

```diff
@@ -17,6 +17,9 @@ var restMethods = {
   'onDemand': {
     'dashboard'     : '/dashboard',
     'issueStatus'   : {method: '/issue/{issueIdOrKey}', queryparams:{fields: ['status']}},
+    'issueUpdate'   : {method: '/issue/{issueIdOrKey}', httpMethod: 'put'},
```

Comment:
Three new JIRA API methods
issueUpdate posts data to update fields in a ticket.
issueTransitions gets data about which statuses a jira issue can be moved to and the ID's of the transitions
issueTransitionUpdate posts data to move an issue from one state to another